app/static/js/events/scheduler.js

```diff
@@ -550,10 +550,32 @@ function addInfoBox($sessionElement, session) {
         html: true,
         title: session.title
     });
-    var speakers = _.map(session.speakers, 'name');
     var content = "";
-    if(speakers.length > 0) {
-        content += "By " + _.join(speakers, ', ') + "<br><br>";
+    if(!_.isNull(session.short_abstract)) {
+        content +=  "<strong>About the session:</strong> " + session.short_abstract + "<br><br>";
+    }
+    if(session.speakers.length > 0) {
+        _.forEach(session.speakers,function(speaker,index) {
+            if(session.speakers.length === 1) {
+                content += "<strong>Speaker: </strong> " + speaker.name + "<br><br>";
+            }
+            else {
+                content += "<strong>Speaker </strong> " + (parseInt(index,10)+1) + "<strong> :</strong> " + speaker.name + "<br><br>";
+            }
+            if(speaker.short_biography) {
+                content += "<strong>About the Speaker: </strong><br>" + speaker.short_biography + "<br><br>";
+            }
+            else {
```

Comment:
Please follow proper code-style ... Keep the else in the same line as `}` .. Like `} else {`